modules/ROOT/pages/addition/instance-requirements.adoc

```diff
@@ -1,8 +1,6 @@
 = Neo4j instance requirements
 
-In order to be managed by NOM, a Neo4j instance **must**:
-
-* be running a Neo4j Enterprise Edition v4.4.0 or higher (including all 5.x versions).
+In order to be managed by NOM, a Neo4j instance **must** be running a Neo4j Enterprise Edition v4.4.0 or higher (including all 5.x versions).
```

Comment:
```suggestion
In order to be managed by NOM, a Neo4j instance **must** be running a Neo4j Enterprise Edition v4.4.2 or higher (including all 5.x versions).
```

Reply:
We should have updated that ages ago. At least in the compatibility matrix it was right